torchvision/models/optical_flow/_raft/raft.py

```diff
@@ -0,0 +1,538 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from torch.nn.modules.batchnorm import BatchNorm2d
+from torch.nn.modules.instancenorm import InstanceNorm2d
+from torchvision.ops import ConvNormActivation
+
+from .utils import grid_sample, make_coords_grid, upsample_flow
+
+
+class ResidualBlock(nn.Module):
+    # This is pretty similar to resnet.BasicBlock except for one call to relu, and the bias terms
```

Comment:
We can probably merge this with the resnet implementation (same for BottleneckBlock below)... but this might make the API awkward and bloated. For now I'd say it's simpler and safer to keep these implementations separate

Reply:
FYI: Agreed. One more reason why we shouldn't expose this on the `__all__`.

Reply:
Let's keep this here as is